docs/user/resources/02-10-sap-btp-operator-cr.md

```diff
@@ -46,29 +46,35 @@ You can't configure any entry parameters in the BtpOperator CR.
+| 16  | Error      | Ready          | false            | ChartPathEmpty                                    | No chart path available for processing                                              |
+| 17  | Error      | Ready          | false            | ConsistencyCheckFailed                            | Failure during consistency check                                                    |
+| 18  | Error      | Ready          | false            | DeletionOfOrphanedResourcesFailed                 | Deletion of orphaned resources failed                                               |
+| 19  | Error      | Ready          | false            | GettingConfigMapFailed                            | Getting ConfigMap failed                                                            |
+| 20  | Error      | Ready          | false            | InconsistentChart                                 | Chart is inconsistent. Reconciliation initialized                                   |
+| 21  | Error      | Ready          | false            | InvalidSecret                                     | `sap-btp-manager` secret does not contain required data - create proper secret      |
+| 22  | Error      | Ready          | false            | PreparingInstallInfoFailed                        | Error while preparing installation information                                      |
+| 23  | Error      | Ready          | false            | ProvisioningFailed                                | Provisioning failed                                                                 |
+| 24  | Error      | Ready          | false            | ReconcileFailed                                   | Reconciliation failed                                                               |
+| 25  | Error      | Ready          | false            | ResourceRemovalFailed                             | Some resources can still be present due to errors while deprovisioning              |
+| 26  | Error      | Ready          | false            | StoringChartDetailsFailed                         | Failure of storing chart details                                                    |
+| 27  | Error      | Ready          | false            | GettingDefaultCredentialsSecretFailed             | Getting default credentials secret failed                                           |
+| 28  | Error      | Ready          | false            | AnnotatingSecretFailed                            | Annotating the required secret failed                                               |
+| 29  | Error      | Ready          | false            | GettingSapBtpServiceOperatorConfigMapFailed       | Getting SAP BTP service operator ConfigMap failed                                   |
+| 30  | Error      | Ready          | false            | GettingSapBtpServiceOperatorClusterIdSecretFailed | Getting SAP BTP service operator Cluster ID secret failed                           |
```

Comment:
```suggestion
| 30  | Error      | Ready          | false            | GettingSapBtpServiceOperatorClusterIdSecretFailed | Getting SAP BTP service operator Cluster ID Secret failed                           |
```

Reply:
As no full stops are used in the other descriptions, let's use a semicolon here.

Reply:
Suggestion applied.